bulbs/liveblog/views.py

```diff
@@ -1,23 +1,26 @@
 from django.conf import settings
 from django.db.models.loading import get_model
-from django.shortcuts import get_object_or_404
-from django.utils import dateparse
+from django.utils import timezone
 from django.views.decorators.cache import cache_control
-from django.views.generic import TemplateView
+
+from bulbs.content.views import BaseContentDetailView
 
 LiveBlogModel = get_model(settings.BULBS_LIVEBLOG_MODEL)
 
 
-class LiveblogNewEntriesView(TemplateView):
+class LiveblogNewEntriesView(BaseContentDetailView):
+    model = LiveBlogModel
     template_name = 'liveblog/new_entries.html'
+    redirect_correct_path = False
 
-    def get_context_data(self, slug, pk):
+    def get_context_data(self, object):
         context = {}
-        liveblog = get_object_or_404(LiveBlogModel, pk=pk)
-        when = dateparse.parse_datetime(self.request.GET['new_as_of'])
-        if not when:
-            raise ValueError('param new_as_of must be an ISO formatted timestamp')
-        context['entries'] = liveblog.entries.filter(published__gte=when)
+        if 'entry_ids' not in self.request.GET:
+            raise ValueError('param "entry_ids" MUST be specified')
+        parsed_entry_ids = [x.strip() for x in self.request.GET['entry_ids'].split(',')]
+        context['entries'] = self.object.entries.filter(
+            pk__in=parsed_entry_ids,
+            published__lte=timezone.now())
```

Comment:
Does this also filter when "published=None" ? (my None comparison logic needs refresher)